modules/KalturaSupport/components/dualScreen/dualScreenControlBar.js

```diff
@@ -121,6 +110,63 @@
 			//Set tooltips
 			buttons.attr('data-show-tooltip', true);
 			this.embedPlayer.layoutBuilder.setupTooltip(buttons, "arrowTop");
+
+			// listen to state change and update buttons style accordingly
+			_this.bind( 'dualScreenStateChange', function(e, state){
```

Comment:
@OrenMe this function is used to listen to changes in player view state and update the UI accordingly (disable/enable icons)